modules/actions.js

```diff
@@ -1,52 +1,78 @@
-// eslint-disable-next-line  no-unused-vars
-const data = JSON.parse(localStorage.getItem('todo-list'));
-const list = document.querySelector('.list');
-const tasks = [
-  {
-    description: 'Finishing project',
-    completed: false,
-    index: 0,
-  }, {
-    description: 'Learning tasks',
-    completed: false,
-    index: 1,
-  }, {
-    description: 'Approving project',
-    completed: false,
-    index: 2,
-  },
-];
+/* eslint-disable max-classes-per-file */
+let tasks = [];
 
-export default class Actions {
-  static showTask(data) {
-    let li = '';
-    if (data) {
-      data.forEach((toDo, id) => {
-        li += `<li class='task'>
-      <input type='checkbox' id='${id}'>
-      <p>${toDo.description}</p>
-      <img class="dots" src="91b1f318fbc3ef6f0456.png" alt="3 dots" style="cursor:pointer"/>
-      </li>
-      `;
+export default class Todo {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+export class Actions {
+  static sincronizationStorage() {
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static addTask() {
+    if (localStorage.getItem('tasks') !== null) {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+      const input = document.querySelector('.list-create input').value;
+      const completed = false;
+      const index = tasks.length;
+      const task = new Todo(input, completed, index);
+      tasks.push(task);
+      document.querySelector('.list-create input').value = '';
+      Actions.sincronizationStorage();
+    } else {
+      tasks = [];
+      const input = document.querySelector('.list-create input').value;
+      const completed = false;
+      const index = 0;
+      const task = new Todo(input, completed, index);
+      tasks.push(task);
+      Actions.sincronizationStorage();
+    }
+  }
```

Comment:
- You did a great job, your design looks nice but make sure you can be able to add tasks, remove tasks, make them complete, add them to the local storage and clear all, on my side that functionality is not working as expected kindly fix that. :+1:

Reply:
Hello, I am sorry,  I think you are reviewing the wrong branch. I already implemented everything in this branch from the given list.